Lib/test/test_marshal.py

```diff
@@ -222,7 +222,10 @@ def test_recursion_limit(self):
         # Create a deeply nested structure.
         head = last = []
         # The max stack depth should match the value in Python/marshal.c.
-        if os.name == 'nt' and hasattr(sys, 'gettotalrefcount'):
+        # BUG: https://bugs.python.org/issue33720
+        # Windows always limits the maximum depth on release and debug builds
+        #if os.name == 'nt' and hasattr(sys, 'gettotalrefcount'):
```

Comment:
No need to mark it as a temporary fix, I suggest:

```
# Windows always limits the maximum depth on release and debug builds
if os.name == 'nt': 
```

Reply:
But I want it to be a temporary fix :) I want the compiler bug that's generating bad stack frames to be fixed. Hopefully I'll hear back from them today whether it's really a bug or not though - if there's a good reason for it then this can become a permanent change.